apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingDao.kt

```diff
@@ -161,7 +160,7 @@ class BehandlingDao(private val connection: () -> Connection) {
                 UPDATE behandling
                 SET status = '${BehandlingStatus.VILKAARSVURDERT}'
                 WHERE status not in (${
```

Comment:
```suggestion
                WHERE status <> ALL (?)
                RETURNING id, sakId
            """.trimIndent()
            stmt.setArray(1, connectionEllerWhaterer.makeArrayEllerNoeIDenDuren("text", BehandlingStatus.skalIkkeOmregnesVedGRegulering().toTypedArray()))
```

denne anbefalingen er garantert å ikke kjøre uten litt tilpasning :D

Reply:
Er ingen endringer som gjorts på det der i denne PR:en. Men uansett hva er forskjellen? Er tester på alla statuser at de blir riktige i `BehandlingDaoReguleringTest.kt´ og har funket prima hittills. 😊

Reply:
så bare ignorer egentlig :D når jeg får tenkt meg om

Reply:
Ser at forslaget du gav er sån det har blitt gjort andre steder så syns egentligen det er grunn nok til å endre sån at vi er konsekvente.